go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/RobustPerception/azure_metrics_exporter
```

Comment:
Is this file needed ???

Reply:
The commited vendor directory is still using govendor. I had to update the go prometheus client version to a later version and the most current way to do it is with go modules. This is a proposition to include in the repository for people who are like me and have moved on from using govendor

Reply:
Ok right !